lib/constable_web/templates/announcement/_comment.html.eex

```diff
@@ -7,11 +7,10 @@
   <div class="media-body">
     <div class="author-information">
       <h4 class="author"><%= @comment.user.name %></h4>
-      <time class="comment-time">
-        <a href="#comment-<%= @comment.id %>">
-          <%= time_ago_in_words @comment.inserted_at %>
```

Comment:
Do you know if we can remove the `time_ago_in_words` function if we're no longer using it? It might just be dead code after this change.

Reply:
I think between this and the other one we might be done with it ... I'll prob merge that one (email one) first and then see if I can kill it here.